### DISCUSSION TITLE
Doesnt accept hotspot address

### DISCUSSION THREAD

**Discussion by jimy619** [davetapley/helium-tax#17] [Q&A] [answered]
First off ..I'm using the long alpha numeric starting with 1..
Eg.  13ZZFMfMKQCmHxGAkQChPDXb39R4tsqCm8TogY85KV4Ndi34F7Z  which is yours..and have tried many others 

Just get the error icon..couldn't find address.. etc.. 

Is it me ? Or.. is there another address which I can't seem to find on helium explorer

**Comment by davetapley** [comment, marked as answer]
Your `13ZZFMfM...` example is an **account** address, not a hotspot address.
An account is something you can send/receive HNT to, and may _own_ many (or zero) hotspots.

It is confusing because they both addresses have the same format 🤪 

Searching on https://explorer.helium.com/ is a good way to check.
A hotspot address will look like this:

<img width="1011" alt="Real_Gunmetal_Tiger___Hotspot_—_Helium_Explorer" src="https://user-images.githubusercontent.com/48232/111036002-45144e80-83da-11eb-8c05-5c0c2c4eb665.png">

---

An account will look like this:

<img width="818" alt="13ZZF______Account_—_Helium_Explorer" src="https://user-images.githubusercontent.com/48232/111035918-d2a36e80-83d9-11eb-820f-2f9bc7a0686b.png">

**Reply by davetapley** [reply]
Additionally there is a request open to be able to run on account address: https://github.com/davetapley/helium-tax/issues/4, but it hasn't done yet.

**Comment by jimy619** [comment]
Aha..thanx..the issue is I use my phone (mobile site) which doesn't show the hotspot address or a few other things..but if I open I to desktop site I can see the hotspot address.

Thak you for clearing this up

⁣Get BlueMail for Android ​

**Reply by davetapley** [reply]
Ah, that's good to know, thanks 🙏🏻

**Comment by piot-admin** [comment]
I've tried a few wallet addresses (including yours lol) but it keeps giving me this error - is it still bec of the outage?
![image](https://user-images.githubusercontent.com/91919101/142475858-aec9b972-052a-495c-8fc0-9baa321fb08c.png)

**Reply by davetapley** [reply]
@piot-admin yeah I wouldn't trust anything until the API is back and stable 😞 

Let me know if the error occurs and https://status.helium.com/  says ✅
